src/style.css

```diff
@@ -0,0 +1,101 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  max-width: 200px;
+  background-color: #fff;
+  font-family: Roboto, sans-serif;
```

Comment:
put the styles for font-family in body tag, so that you don't need to specify them for each element you add on the page